appshell/appshell_extensions_platform.h

```diff
@@ -79,6 +80,31 @@ inline void* getMenuParent(CefRefPtr<CefBrowser>browser) {
 #endif
 
 
+class CharSetDetect
+{
+	UCharsetDetector* charsetDetector_;
```

Comment:
convention is to start a member with m, should you be prefixing data members with `m`?